rocAL/source/api/rocal_api_augmentation.cpp

```diff
@@ -426,6 +426,51 @@ rocalCropResizeFixed(
     return output;
 }
 
+RocalTensor ROCAL_API_CALL
+rocalROIResize(
+    RocalContext p_context,
+    RocalTensor p_input,
+    unsigned dest_width,
+    unsigned dest_height,
+    bool is_output,
+    unsigned roi_h,
+    unsigned roi_w,
+    float roi_pos_x,
+    float roi_pos_y,
+    RocalResizeInterpolationType interpolation_type,
+    RocalTensorLayout output_layout,
+    RocalTensorOutputType output_datatype) {
+    Tensor* output = nullptr;
+    if ((p_context == nullptr) || (p_input == nullptr)) {
+        ERR("Invalid ROCAL context or invalid input tensor")
+        return output;
+    }
+    auto context = static_cast<Context*>(p_context);
+    auto input = static_cast<Tensor*>(p_input);
+    try {
+        if (dest_width == 0 || dest_height == 0)
+            THROW("ROI Resize needs to receive non-zero destination dimensions")
+
+        RocalTensorlayout op_tensor_layout = static_cast<RocalTensorlayout>(output_layout);
+        RocalTensorDataType op_tensor_datatype = static_cast<RocalTensorDataType>(output_datatype);
+        TensorInfo output_info = input->info();
+        output_info.set_data_type(op_tensor_datatype);
+
+        // For the ROI resize augmentation, user can create an image with a different width and height
+        output_info.modify_dims_width_and_height(op_tensor_layout, dest_width, dest_height);
```

Comment:
please rename to "set_new_dims()"

Reply:
This is an existing API in tensor.h and used for all augmentation APIs where we are setting the output tensor width and height. The reasoning behind this API name is we are only setting the new width and height. 
In case of numpy reader where we will be using generic ROI, we are using the existing set_dims function in tensor.h to set new dims for the output tensor. 
So the set_new_dims name might not be suitable for the above function which only sets w/h for output tensor

Reply:
please change to set_new_width_and_height instead